xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-filters/xwiki-platform-notifications-filters-api/src/main/java/org/xwiki/notifications/filters/NotificationFilterPreference.java

```diff
@@ -77,7 +77,10 @@ public interface NotificationFilterPreference
      * notifications.
      *
      * @return true if the filter preference is active.
+     * @deprecated this behaviour doesn't make sense anymore with usage of prefiltering as there's no trigger for
+     * retrieving the notifications nowadays.
      */
+    @Deprecated(since = "16.5.0RC1")
     boolean isActive();
```

Comment:
Do you plan to move this to legacy?

Reply:
Yes I started to work on providing legacy interface as part of #2882 the plan would be to move that method to legacy too once the other PR is merged.

Reply:
Actually I can't move it yet to legacy: the column `isActive` is exposed in hibernate configuration so hibernate needs a getter for it and moving to legacy means that it would be lost in standard docker tests leading to errors. 
I will add a note in javadoc. 
Now note that my goal is to refactor the hibernate mapping for filters as part of #3154 so I should be able to do it in there.